app/controllers/reservation_frames_controller.rb

```diff
@@ -36,4 +36,18 @@ def create
 
     redirect_to new_reservation_frame_path, notice: !flash[:alert] && "登録しました"
   end
+
+  def update
+    if ReservationFrame.find(params[:id]).status_ok?
+      ReservationFrame.find(params[:id]).update(status: "ng")
+    else
+      ReservationFrame.find(params[:id]).update(status: "ok")
+    end
```

Comment:
`ReservationFrame.find(params[:id])`が何度も登場してるから一旦変数に格納したい。

```suggestion
   reservation_frame = ReservationFrame.find(params[:id])
    if reservation_frame.status_ok?
      reservation_frame.update(status: "ng")
    else
      reservation_frame.update(status: "ok")
    end
```

Reply:
あとenum使っている場合だと`status_ng!`とかで更新もできるはず！
（もしかしたら`ng!`だけかもしれないけど）

```ruby
reservation_frame.status_ng!
```

Reply:
確かにそうですね やってみます！